README.md

```diff
@@ -1,2 +1,35 @@
-# awesome-books
-A basic website that allows users to add/remove books from a list.
+# Awesome Books Collection
+
+> A basic website that allows users to add/remove books from a list.
+
+## Built With
+
+- HTML, CSS, Javascript
+
+👤 **Yasin**
+
+- GitHub: [@Yazino12](https://github.com/Yazino12)
+
+👤 **Fahd Rahali**
+
+- GitHub: [@fahdrahali](https://github.com/fahdrahali)
```

Comment:
- [ ] Kindly create an `Authors` section and add the `Authors` to that section and remove it from the `Built With` section.
- [ ] Kindly add the `Getting Started` section and add instructions on how to clone your repository and set it up in the terminal.

Reply:
Hello @julie-ify I accidently closed this pull request and I am afraid that I cannot re-open it, and the link I shared on the dashboard is this pull request. I have created another pull request with the changes on this branch. If it's possible to re-open this pull request, how can I do that?